install/README.md

```diff
@@ -0,0 +1,8 @@
+# hls-install
+
+This directory contains the `hls-install` project, an Haskell Language Server installer.
+It is used when [installing from sources](../docs/installation.md#building).
```

Comment:
We are using absolute urls to readthedocs for links, see the main README as example

Reply:
Done (and I took the liberty to fix a few other links in the documentation)

Reply:
oh, but links inside docs themselves can be relative, we need absolute ones in external resources like README's

Reply:
Oh I see, my bad :facepalm: I have force-pushed the fix

Reply:
dont worry, i could be more specific, will update to head to get the ci fix